src/target/stm32f1.c

```diff
@@ -254,6 +256,91 @@ static bool at32f41_detect(target_s *target, const uint16_t part_id)
 	return true;
 }
 
+static bool at32f43_detect(target_s *target, const uint16_t part_id)
+{
+	/* AT32F435 EOPB0 ZW/NZW split reconfiguration unsupported,
```

Comment:
The opening line of a multi-line comment should only contain the `/*` sequence, with the first textual line on the next line as in
```c
/*
 * A Multi-line
 * comment
 */
 ```

As this is cosmetic and the code base still needs harmonisation done on it for this, we'll let it pass in this case though